### DISCUSSION TITLE
I have an question

### DISCUSSION THREAD

**Discussion by bhaveshjadav713** [twbs/bootstrap#39171] [Q&A] [answered]
I have an question.

I want to build and sell my template, is there any time period to approve seller account?

**Comment by julien-deramond** [comment, marked as answer]
Hello @bhaveshjadav713 
I suppose that you're talking about https://themes.getbootstrap.com/. This project is not handled here in GitHub and is not managed by the Bootstrap core team itself.
I suggest that you ask this question directly in their Help Center.

**Comment by bhaveshjadav713** [comment]
Thanks for your reply.
